components/navbars.py

```diff
@@ -90,7 +90,8 @@ class RegistriesNavbar(EmberNavbar):
     user_dropdown_support = Locator(By.CSS_SELECTOR, 'ul.dropdown-menu-right > li:nth-child(2) > a')
     user_dropdown_settings = Locator(By.CSS_SELECTOR, 'a[data-analytics-name="Settings"]')
 
-    add_new_link = Locator(By.CSS_SELECTOR, 'a[data-test-add-new-button]')
+    add_new_link = Locator(By.CSS_SELECTOR, 'a[data-analytics-name="Create new registration"]')
```

Comment:
the `data-test-*` attrs are precisely intended for writing tests like these -- curious why the switch to `data-analytics-name`?

Reply:
The `data-test-add-new-button` was too generic because it was being used to describe 2 different elements on the page.  But you bring up a good point! Instead of using analytics, I've added the `href` attribute to specify which button we are clicking on in the navbar.